### ISSUE TITLE
Use a Plack middleware for supporting FilterContent output filters

### ISSUE THREAD

**Issue by bschmalhofer** [RotherOSS/otobo#737]
This is more of a project idea. In OTOBO there are output filters that modify the generated Response. In Plack there are middlewares that do the same thing. So the idea is to implement the OTOBO output filters as one, or several??, Plack midddlewares. This looks like a cleaner architecture.

Note that this would only apply to the filters that are configured as 'FilterConten'.

But first we need to check whether any features would be lost.

**Comment by bschmalhofer** [comment]
I sounded like a good idea, but it didn't pan out. `$Kernel::OM` is localised in the handle `$OTOBOApp`. For putting the FilterContent feature we would have to expand the scope of `$Kernel::OM` or create a separate instance in the middleware. Both workaraounds are not nice. 

In order to make at least some progress I have moved the call to ApplyOutputHandlers into  `$OTOBOApp` . This simplifies the interface modules a little bit.
Also: clean up the redundant return statement in the interface modules.

The issue will be closed when rel-10_1 looks stable again.
